package.json

```diff
@@ -88,12 +92,12 @@
     "typescript": "3.7.5"
   },
   "engines": {
-    "node": ">=10.9.0 <11.0.0",
-    "npm": ">=5.0.0"
+    "node": ">=12 <13",
+    "yarn": ">=1.22.4 <2"
```

Comment:
I would also add a:
```
    "npm": "____PLEASE_USE_YARN____",
```
And with this `.npmrc ` file:
```
engine-strict=true
```

You will get this error on `npm install`:
<details>
<summary>click to view log</summary>

```
> npm install
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for MY_PROJECT@0.0.0-dev: wanted: {"node":">=12 <13","npm":"____PLEASE_USE_YARN____","yarn":">=1.22.4 <2"} (current: {"node":"12.16.1","npm":"6.14.5"})
npm ERR! notsup Not compatible with your version of node/npm: MY_PROJECT@0.0.0-dev
npm ERR! notsup Not compatible with your version of node/npm: MY_PROJECT@0.0.0-dev
npm ERR! notsup Required: {"node":">=12 <13","npm":"____PLEASE_USE_YARN____","yarn":">=1.22.4 <2"}
npm ERR! notsup Actual:   {"npm":"6.14.5","node":"12.16.1"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/MY_USER/.npm/_logs/2020-06-04T14_50_00_430Z-debug.log

```

</details>

Reply:
Interesting suggestion, I'll try it out!